docs/source/integrations/databricks_development_workflow.md

```diff
@@ -0,0 +1,324 @@
+
+```bash
+git branch feat/modified-classifier
+git checkout feat/modified-classifier
+```
+
+#### Modify the classifier
+
+The PySpark Iris starter uses a one-nearest neighbour classifier to make predictions. In this section, you will modify it to create predictions using a three-nearest neighbour classifier. A three-nearest neighbour classifier works by finding the most common class among the three closest neighbours to make a prediction. A one-nearest neighbour classifier only uses the class of the nearest neighbour.
+
+Open the file `<project_root>/src/iris_databricks/nodes.py` in your IDE. Insert a new line underneath the import statements and add the following code to it:
+
+```python
+from scipy.stats import mode
+```
+
+Now modify the classifier by changing the code in the function `make_predictions`:
+
+```python
+def make_predictions(
+    X_train: pd.DataFrame, X_test: pd.DataFrame, y_train: pd.DataFrame
+) -> pd.DataFrame:
+    """Uses 3-nearest neighbour classifier to create predictions.
+    Args:
+        X_train: Training data of features.
+        y_train: Training data for target.
+        X_test: Test data for features.
+    Returns:
+        y_pred: Prediction of the target variable.
+    """
+
+    X_train_numpy = X_train.to_numpy()
+    X_test_numpy = X_test.to_numpy()
+
+    squared_distances = np.sum(
+        (X_train_numpy[:, None, :] - X_test_numpy[None, :, :]) ** 2, axis=-1
+    )
+    three_nearest_neighbours = np.argsort(squared_distances, axis=0)[:3]
+
+    # Use list comprehension to index y_train for each test point's three nearest neighbours
+    y_nearest_neighbours = np.array(
+        [y_train.iloc[three_nearest_neighbours[:, i]] for i in range(X_test.shape[0])]
+    )
+
+    y_pred_mode, _ = mode(y_nearest_neighbours, axis=1)
+    y_pred = pd.DataFrame(
+        y_pred_mode.squeeze(), columns=y_train.columns, index=X_test.index
+    )
+
+    return y_pred
+```
+
+The next step is to commit the modified code to your repository. Check that you have the `feat/modified-classifier` branch checked out using the following command:
+
+```bash
+git branch
+```
+
+You should see the following output:
+
+```bash
+* feat/modified-classifier
+  main
+```
+
+The `*` character next to `feat/modified-classifier` indicates that this is the currently checked-out branch. Now commit your changes to the branch and push it to your remote repository:
+
+```bash
+git add -u
+git commit -m "Modify 1-nearest neighbour classifier to 3-nearest-neighbour classifier"
+git push origin --set-upstream feat/modified-classifier
+```
+
+#### Pull the new branch and check it out on Databricks Repos
+
+Navigate to your Databricks Repo in the Databricks Workspace UI. To pull the changes from your remote repository, look for the `Pull` button in the upper-right corner of the Repo page. Click it to sync the latest changes from the remote repository into your Databricks Repo.
```

Comment:
It would be useful to add a picture here of the pull button and it's position you mention.